Author: Mike Markley <mike@markley.org>
Author: Sergei Golovan <sgolovan@debian.org>
Description: Fixes shebangs in examples (Closes: #152367).

--- a/example/beer.exp
+++ b/example/beer.exp
@@ -1,4 +1,9 @@
-#!/depot/path/expect -f
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec tclsh "$0" ${1+"$@"}
+
+package require Expect
 
 # 99 bottles of beer on the wall, Expect-style
 # Author: Don Libes <libes@nist.gov>
--- a/example/expectd.proto
+++ b/example/expectd.proto
@@ -1,4 +1,10 @@
-#!/depot/tcl/src/expect/e --
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec tclsh "$0" ${1+"$@"}
+
+package require Expect
+
 # Description: Simple fragment to begin a telnet daemon
 # For more information, see Chapter 17 of "Exploring Expect"
 # Author: Don Libes, NIST
--- a/example/irsh
+++ b/example/irsh
@@ -1,4 +1,9 @@
-#!/depot/path/expect --
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec tclsh "$0" ${1+"$@"}
+
+package require Expect
 
 # Do rsh interactively.  For example, consider the following command:
 #    rsh <remote> ls -l "|" more
--- a/example/passwd.cgi
+++ b/example/passwd.cgi
@@ -1,4 +1,9 @@
-#!/depot/path/expect --
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec tclsh "$0" ${1+"$@"}
+
+package require Expect
 
 # This is a CGI script to process requests created by the accompanying
 # passwd.html form.  This script is pretty basic, although it is
--- a/example/passwdprompt
+++ b/example/passwdprompt
@@ -1,4 +1,9 @@
-#!/depot/path/expect
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec tclsh "$0" ${1+"$@"}
+
+package require Expect
 
 # This script prompts for a passwd from stdin while echoing *'s
 
--- a/example/reprompt
+++ b/example/reprompt
@@ -1,4 +1,9 @@
-#!/depot/path/expect --
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec tclsh "$0" ${1+"$@"}
+
+package require Expect
 
 # Name: reprompt
 # Description: reprompt every so often until user enters something
--- a/example/term_expect
+++ b/example/term_expect
@@ -1,4 +1,9 @@
-#!/depot/path/expectk
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec wish "$0" ${1+"$@"}
+
+package require Expect
 
 # Name: tkterm - terminal emulator using Expect and Tk text widget, v3.0
 # Author: Don Libes, July '94
--- a/example/vrfy
+++ b/example/vrfy
@@ -1,4 +1,9 @@
-#!/depot/path/expect -f
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec tclsh "$0" ${1+"$@"}
+
+package require Expect
 
 
 # separate address into user and host
--- a/example/xrlogin
+++ b/example/xrlogin
@@ -1,4 +1,10 @@
-#!/depot/path/expect --
+#!/bin/sh
+# -*- tcl -*-
+# The next line is executed by /bin/sh, but not tcl \
+exec tclsh "$0" ${1+"$@"}
+
+package require Expect
+
 # xrlogin - rlogin but with current DISPLAY
 #
 # You can extend this idea to save any arbitrary information across rlogin
