Author: n/a
Description: Fixes some unsafe traps.

--- expect-5.44.1.15.orig/exp_trap.c
+++ expect-5.44.1.15/exp_trap.c
@@ -261,6 +261,18 @@
 #if defined(SIGSTOP)
 	traps[SIGSTOP].reserved = TRUE;
 #endif
+#if defined(SIGSEGV)
+	traps[SIGSEGV].reserved = TRUE;
+#endif
+#if defined(SIGBUS)
+	traps[SIGBUS].reserved = TRUE;
+#endif
+#if defined(SIGILL)
+	traps[SIGILL].reserved = TRUE;
+#endif
+#if defined(SIGFPE)
+	traps[SIGFPE].reserved = TRUE;
+#endif
 
 	async_handler = Tcl_AsyncCreate(tophalf,(ClientData)0);
 
